Wordle.Api/Wordle.Api.Tests/PlayerServiceTests.cs

```diff
@@ -5,40 +5,73 @@
 using System.Text;
 using System.Threading.Tasks;
 using Wordle.Api.Data;
+using Wordle.Api.Dtos;
 using Wordle.Api.Services;
 
-namespace Wordle.Api.Tests
+namespace Wordle.Api.Tests;
```

Comment:
You have a bunch of formatting changes - this make your git diff difficult to read. Make sure the changes you make to your assignment are relevant to the assignment.